app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
```

Comment:
After the opening bracket, write all arguments on a new line, and write the closing bracket on a new line as well.

Reply:
Done